sdk/identity/Azure.Identity/src/Credentials/EnvironmentCredential.cs

```diff
@@ -154,6 +156,8 @@ public override AccessToken GetToken(TokenRequestContext requestContext, Cancell
         /// <param name="requestContext">The details of the authentication request.</param>
         /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
         /// <returns>An <see cref="AccessToken"/> which can be used to authenticate service client calls, or a default <see cref="AccessToken"/>.</returns>
+        /// <exception cref="AuthenticationFailedException">Thrown when the authentication failed.</exception>
+        /// <exception cref="CredentialUnavailableException">Thrown when the credential is unavailable when the environment is not properly configured.</exception>
```

Comment:
```suggestion
        /// <exception cref="CredentialUnavailableException">Thrown when the credential is unavailable because the environment is not properly configured.</exception>
```